test/view_component/integration_test.rb

```diff
@@ -360,10 +360,10 @@ class IntegrationTest < ActionDispatch::IntegrationTest
 
     assert_select(".footer.text-blue h3", text: "This is the footer")
 
-    title_node = Nokogiri::HTML.fragment(response.body).css(".title").to_html
-    expected_title_html = "<div class=\"title\">\n    <p>This is my title!</p>\n  </div>"
+    title_node = Nokogiri::HTML.fragment(response.body).css(".title").to_html.strip
+    expected_title_html = "<div class=\"title\"><p>This is my title!</p></div>"
 
-    assert_equal(title_node, expected_title_html)
+    assert_equal(expected_title_html, title_node)
```

Comment:
Swapped these, `assert_equal(expected, actual)` the test suite was calling them this way and it was confusing.